<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>1620</width>
      <height>780</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>24</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>System Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Open Price 1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>High Price 1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Low Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>High Low Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>%K</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>BUY Target</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SELL Target</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MA Score</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Contracts Ready</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
